api/candlepin-api-spec.yaml

```diff
@@ -193,6 +193,50 @@ paths:
               schema:
                 $ref: '#/components/schemas/ExceptionMessage'
 
+  /deleted_consumers:
+    get:
+      description: Retrieves a list of Deleted Consumers By deletion date or all. List returned is the deleted Consumers.
+      tags:
+        - Consumer
+        - DeletedConsumer
+      operationId: listByDate
+      parameters:
+        - in: query
+          name: date
+          schema:
+            type: string
+          description: ''
```

Comment:
Maybe something like "filter deleted consumers to those on or after this date"?

Reply:
@kahowell 
Added description value in the spec. Please re-review.